build/appveyor/before_build.bat

```diff
@@ -22,12 +22,12 @@ python build/add-mc-keys.py %MC_CONSUMER_KEY% %MC_CONSUMER_SECRET%
 )
 
 :: get revision number
-SET PATH=C:\Qt\5.4\mingw491_32\bin;C:\Qt\Tools\mingw491_32\bin;%PATH%
+SET PATH=C:\Qt\5.9.6\mingw53_32\bin;%PATH%
 mingw32-make -f Makefile.mingw revision
 SET /p MSversion=<mscore\revision.h
 
 :: CMake refuses to generate MinGW Makefiles if sh.exe is in the PATH (C:\Program Files\Git\usr\bin)
-SET PATH=C:\Qt\5.4\mingw491_32\bin;C:\Qt\Tools\mingw491_32\bin;C:\Program Files (x86)\CMake\bin;C:\Program Files\7-Zip;C:\ccache\bin;C:\Tools\curl\bin;%WIX%\bin;C:\Windows\system32;C:\Windows
+SET PATH=C:\Qt\5.9.6\mingw53_32\bin;C:\Qt\Tools\mingw53_32\bin;C:\Program Files (x86)\CMake\bin;C:\Program Files\7-Zip;C:\ccache\bin;C:\Tools\curl\bin;%WIX%\bin;C:\Windows\system32;C:\Windows

```

Comment:
`C:\Qt\Tools\mingw53_32\lib here too

Reply:
Should I be adding C:\Qt\Tools\mingw53_32\lib to PATH or editing C:\Qt\Tools\mingw53_32\bin; to C:\Qt\Tools\mingw53_32\lib;?

Reply:
Oh boy, fat fingers again, sorry, I meant `C:\Qt\Tools\mingw53_32\bin`

Reply:
No problem :D

Reply:
feels like we have no change to be done here

Reply:
Unfortunatly we do, as I screwed up again,  I meant `C:\Qt\Tools\mingw530_32\bin`

Reply:
It is just way to warm here